deployment/morph/webghc.nix

```diff
@@ -52,15 +51,15 @@
         home = "/home/monitor";
         description = "a user for administering web-ghc";
         extraGroups = [ "systemd-journal" ];
-        openssh.authorizedKeys.keys = machines.playgroundSshKeys;
+        openssh.authorizedKeys.keys = monitoringKeys;
         packages = [ serviceSystemctl ];
       };
 
       systemd.services.web-ghc = {
         wantedBy = [ ];
         before = [ ];
         enable = true;
-        path = [ "${web-ghc}" ];
+        path = [ ];
```

Comment:
You can omit the empty path because `[ ]` is the default value anyhow. Same goes for `wantedBy` and `before`.
What you could maybe *add* though is

```
after = [ "network.target" ];
```